apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -244,23 +246,14 @@ onBeforeMount(() => {
 
 const addTokens = () => {
   Object.keys(FORMAT_MAP).forEach(format => {
+    processedTokens[format] = {};
     for (const theme of THEMES) {
-      Object.keys(CATEGORY_MAP).forEach(category => {
-        const tokens = [];
-        Object.entries(tokensJson[theme.value])
-          .filter(([key, value]) => CATEGORY_MAP[category].includes(key.split('/')[0]) && value[FORMAT_MAP.CSS])
-          .forEach(([_, value]) => {
-            const { name, value: tokenValue, description, keywords } = value[FORMAT_MAP[format]] || {};
-            // exclude base tokens
-            if (name && !isBaseToken(name)) {
-              const { value: exampleValue, name: exampleName } = value[FORMAT_MAP.CSS];
-              tokens.push({ exampleValue, exampleName, name, tokenValue, description, keywords });
-            }
-          });
-        if (!processedTokens[format]) processedTokens[format] = {};
-        if (!processedTokens[format][theme.value]) processedTokens[format][theme.value] = {};
-        processedTokens[format][theme.value][category] = [...tokens];
+      // initialize processedTokens with desired structure
+      const structure = getTokensStructure();
+      Object.entries(tokensJson[theme.value]).forEach((token) => {
+        addTokenToStructure(token, format, structure);
       });
+      processedTokens[format][theme.value] = structuredClone(structure);
```

Comment:
if `getTokensStructure` were just a const we could easily use it here, without creating a new const and avoid using `structuredClone`, not sure why we are using it anyways, maybe Im missing some edge case here.

Reply:
I suppose I could assign `getTokensStructure ` directly to `processedTokens[format][theme.value]` and populate that one to avoid a `structuredClone`